### DISCUSSION TITLE
Builder app not providing Vanilla builds when running the project in local

### DISCUSSION THREAD

**Discussion by Barabasbalazs** [webstudio-is/webstudio-community#194] [Q&A] [answered]
I've been tinkering about with the builder app in a local environment for a couple days now, but can't get it no matter what to generate a build for Vanilla projects(Haven't tried the other ones yet). 

![wstd_ss](https://github.com/user-attachments/assets/1151e3b2-3306-440f-9654-53c64352e8c8)

After digging in to the code it seems like the build makes it into the DB (at least the ZIP name shows up):

![psql_ss](https://github.com/user-attachments/assets/5badca83-de71-4f72-b349-761de2a85acd)

It seems though that the publishStatus remains on PENDING which from my understanding means that the build can't be downloaded?

Can somebody clarify whether this functionality should work on a self-hosted version, or is it specified in these issues:

- https://github.com/webstudio-is/webstudio/issues/3966
- https://github.com/webstudio-is/webstudio/issues/4061 

 It seems to work when using  [webstudio.is](url)

**Comment by TrySound** [comment, marked as answer]
We use our cloud to build websites on publish. Though there is no cloud locally. So after clicking "publish" you can use Webstudio CLI to build the project and deploy it whenever you need. Our cloud does basically the same.

**Comment by Barabasbalazs** [comment]
I see. Could we possibly update the docs somehow to reflect this workflow? I did find the intructions about the CLI in its separate section, but maybe It would be useful to add a link to it in the self hosting part of the docs

**Reply by kof** [reply]
I am not sure where you would find the link useful, but please do send us the change where you think it would have helped you. Docs are available for contribution

<img width="160" alt="image" src="https://github.com/user-attachments/assets/e93f9d6b-6381-4707-b3e7-07046c6ecb44">

**Reply by Barabasbalazs** [reply]
will do. Thank you

**Reply by Barabasbalazs** [reply]
https://github.com/webstudio-is/webstudio-community/pull/195
